pydropsonde/circles.py

```diff
@@ -220,7 +253,7 @@ def fit2d_xr(self, x, y, u, sonde_dim="sonde"):
 
     def apply_fit2d(self, variables=None):
         if variables is None:
-            variables = ["u", "v", "q", "ta", "p", "density"]
+            variables = ["u", "v", "q", "ta", "p", "rh", "theta"]
```

Comment:
maybe it's better to use the same loop as above?
```suggestion
            variables = [
                var
                for var in ds.variables
                if set([alt_dim, "sonde_id"]).issubset(ds.variables[var].dims)
            ]:
```

Reply:
Yes, seems consistent. I'll add it to be able to test the final code changes